kview/src/main/java/dev/ikm/komet/kview/mvvm/viewmodel/stamp/StampAddFormViewModelBase.java

```diff
@@ -60,10 +64,19 @@ protected void doUpdate(EntityFacade entity, UUID topic, ViewProperties viewProp
     }
 
     protected void loadStamp() {
-        EntityVersion latestVersion = viewProperties.calculator().latest(entityFacade).get();
-        StampEntity stampEntity = latestVersion.stamp();
+        ObservableEntity observableEntity = ObservableEntity.get(entityFacade.nid());
+        ObservableEntitySnapshot observableEntitySnapshot = observableEntity.getSnapshot(viewProperties.calculator());
+        Latest<EntityVersion> latestEntityVersion = retrieveCommittedLatestVersion(observableEntitySnapshot);
+        latestEntityVersion
+                .ifPresentOrElse(latestVersion -> setPropertyValue(CURRENT_STAMP, latestEntityVersion.get().stamp())
+                    , () -> {
+                               EntityVersion latestVersion = viewProperties.calculator().latest(entityFacade).get();
+                               setPropertyValue(CURRENT_STAMP, latestVersion.stamp());
+                    }
+                );
```

Comment:
Not sure if changing this code could break other Components for which this is also used and was working